src/core/fasta_api.h

```diff
@@ -29,10 +29,29 @@
 void gt_fasta_show_entry(const char *description, const char *sequence,
                          GtUword sequence_length, GtUword width, GtFile *outfp);
 
+/* Print a fasta entry with optional <description> and mandatory <sequence> to
+   <outfp>. If <width> is != 0 the sequence is formatted accordingly.
+   Will print at most <sequence_length> characters from <sequence> and at most
+   <description_length> characters from <description> if present. */
+void gt_fasta_show_entry_nt(const char *description, GtUword description_length,
+                            const char *sequence, GtUword sequence_length,
+                            GtUword width, GtFile *outfp);
+
 /* Append a fasta entry with optional <description> and mandatory <sequence> to
    <outstr>. If <width> is != 0 the sequence is formatted accordingly. */
 void gt_fasta_show_entry_str(const char *description, const char *sequence,
                              GtUword sequence_length, GtUword width,
                              GtStr *outstr);
 
+/* Print a fasta entry with optional <description> and mandatory <sequence> to
+   <outstr>. If <width> is != 0 the sequence is formatted accordingly.
+   Will print at most <sequence_length> characters from <sequence> and at most
+   <description_length> characters from <description> if present. */
+void gt_fasta_show_entry_nt_str(const char *description,
```

Comment:
There is no `outstr` in the function prototype. Either the prototype is wrong or the function name should be different (no `_str` at the end).